data/json/items/tool/med.json

```diff
@@ -158,7 +158,7 @@
     "id": "vacutainer",
     "type": "CONTAINER",
     "category": "tools",
-    "name": "blood draw kit",
+    "name": { "str": "blood draw kit" },
     "description": "This is a kit for drawing blood, including a test tube for holding the sample.  Use this tool to draw blood, either from yourself or from a corpse you are standing on.",
     "weight": "13 g",
     "volume": "250 ml",
```

Comment:
this seems way to much for normal blood draw kit.

Reply:
The volume?i did not change that. Im just changing the name. It will have to be resoved in another pr.